web/html/media/video-formats.md

```diff
@@ -35,8 +35,6 @@ As you can see, you really only need 2 HTML5 video formats for modern web browse
 
 This CodePen will provide all the necessary code - it's a decent chuck compared to just the `video` element.
 
-[View CodePen](https://codepen.io/enkidevs/pen/xJbYLE)
```

Comment:
@nem035 Should we prefer this codepen button over link, coz we have a custom button for it now?

Reply:
Yeah, I think that makes sense. Is the md syntax the button expects in the form of a md link?

Reply:
yea, a hyperlink at root level is converted into a button

Reply:
The button (as it is now) is always in the middle of the insight and takes the user away from the app in the middle of reading the content, whereas when its a link in the link section it is located at the bottom and users would first finish the insight and then wander away from the app to test what they've learned

Reply:
Valid point, I see.

Reply:
Update: based on our latest discussions, we should put this button back

Reply:
I thought I already posted a PR with the opposite changes done in this one(regarding codepens) and mistakenly never posted that one and thought this one was the new one. I am posting the new one now.